source/common/grpc/codec.cc

```diff
@@ -38,7 +38,6 @@ bool Decoder::decode(Buffer::Instance& input, std::vector<Frame>& output) {
   output_ = &output;
   inspect(input);
   output_ = nullptr;
-
```

Comment:
Nit: unrelated?

Reply:
Thanks!

I actually revert this unrelated change in next pr https://github.com/envoyproxy/envoy/pull/32676.  Can I change it there?(just want to avoid the long CI run if it is ok)

Reply:
Sure

Reply:
When do you plan on adding release notes?

Reply:
Good point. Can I also add it in https://github.com/envoyproxy/envoy/pull/32676 which is for gRPC error code improvement? because i am thinking that is where I complete this effort.

If you prefer rel note to be added here, i am also happy to do so.